src/slider.styles.ts

```diff
@@ -0,0 +1,174 @@
+import { css } from 'lit';
+import focusOutline from './styles/focus-outline.js';
+
+export default [
+  css`
+    ${focusOutline('.handle:focus-visible')}
+  `,
+  css`
+    /* The designs call for a bit more spacing than the default when vertical. */
+    glide-core-private-label[orientation='vertical']::part(private-tooltips) {
+      margin-block-end: var(--glide-core-spacing-base-xxs);
+    }
+
+    .slider-container {
+      --private-track-size: 0.375rem;
+
+      align-items: center;
+      display: flex;
+      gap: var(--glide-core-spacing-base-md);
+      inline-size: 100%;
+      justify-content: space-between;
+    }
+
+    .input {
+      /* stylelint-disable-next-line property-no-vendor-prefix */
+      -moz-appearance: textfield;
+      background-color: var(--glide-core-color-interactive-surface-container);
+      block-size: 2.125rem;
+      border: 1px solid var(--glide-core-color-interactive-stroke-primary);
+      border-radius: var(--glide-core-rounding-base-radius-sm);
+      box-sizing: border-box;
+      inline-size: 2.125rem;
+      text-align: center;
+
+      &::-webkit-outer-spin-button,
+      &::-webkit-inner-spin-button {
+        appearance: none;
+      }
+
+      &:focus,
+      &:hover {
+        border-color: var(--glide-core-color-interactive-stroke-primary--hover);
+        outline: none;
+        transition: border-color 200ms ease-in-out;
+      }
+
+      &.disabled {
+        background-color: var(
+          --glide-core-color-interactive-surface-container--disabled
+        );
+        border-color: var(
+          --glide-core-color-interactive-stroke-primary--disabled
+        );
+        color: var(--glide-core-color-interactive-text-default--disabled);
+      }
+
+      &.error {
+        border-color: var(--glide-core-color-advisory-stroke-error-primary);
+      }
+
+      /*
+        We had to resort to a class selector because there may be a bug in Chrome and Safari
+        with ":read-only": https://bugs.chromium.org/p/chromium/issues/detail?id=1519649
+      */
+      &.readonly {
+        background-color: transparent;
+        border: 1px solid transparent;
+        padding-inline-start: 0;
+      }
+    }
+
+    .track-container {
+      flex: 1;
+      min-inline-size: 0;
+
+      &.single {
+        /*
+            Without this suggestion from design, when the handle
+            is at the min, it would overflow the container and
+            not align with label. This isn't needed in multiple
+            mode because we have an input to help prevent this
+            from happening.
+         */
+        padding-inline-start: var(--glide-core-spacing-base-sm);
```

Comment:
Hopefully this makes sense.

Without the additional padding 👎 

<img width="579" alt="Screenshot 2025-05-20 at 10 28 41 AM" src="https://github.com/user-attachments/assets/3d5b68de-13fc-478c-921d-4b616634649a" />

<img width="580" alt="Screenshot 2025-05-20 at 10 29 03 AM" src="https://github.com/user-attachments/assets/41f1bfcb-d4a0-4dc2-a612-98752dfe7fe7" />

Reply:
They're okay with the error message being misaligned as a result?

<img width="760" alt="image" src="https://github.com/user-attachments/assets/a520e75f-a066-4ad0-9588-17359d4caf16" />

Reply:
Yeah. It looks off with a single slider for sure, but when `multiple` it's not bad because we have the input on the left-hand side too.